src/include/rbd/librbd.h

```diff
@@ -159,6 +159,11 @@ typedef struct {
 } rbd_group_spec_t;
 
 typedef enum {
+  GROUP_SNAP_STATE_PENDING,
```

Comment:
Don't forget to add the C and Python API methods to match the new C++ API methods